src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DATE_FORMAT =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    private DataRow parse(String line) {
```

Comment:
`public String getSalaryInfo` should be first.
![image](https://user-images.githubusercontent.com/20326858/234818915-36a6224b-e71d-409e-9b98-e256ff7f24c7.png)